build/haproxy_check_replicas.sh

```diff
@@ -2,6 +2,13 @@
 
 set -e
 
+log() {
+	local level=$1
+	local message=$2
+
+	echo "$(/usr/bin/date +%Y-%m-%dT%H:%M:%S%z) [${level}] ${message}"
```

Comment:
The support team wants to have JSON format like it was required in PXC. It simplifies there life when you need to parse something.

Reply:
done

Reply:
but I don't really like it since we have logs in mixed format right now:

```
{"time":"28/Jul/2023:06:55:50.971", "level": "INFO", "message": "10.100.1.7:33062 @@super_read_only: 0 Applier: ON"}
{"time":"28/Jul/2023:06:55:50.974", "level": "INFO", "message": "10.100.1.7:33062 for backend mysql-replicas is NOT OK"}
[WARNING]  (19) : Server mysql-replicas/cluster1-mysql-0 is DOWN, reason: External check error, code: 1, check duration: 149ms. 2 active and 0 backup servers left. 0 sessions active, 0 requeu
ed, 0 remaining in queue.
{"time":"28/Jul/2023:06:55:52.629", "level": "INFO", "message": "10.100.2.7:33062 @@super_read_only: 1 Applier: ON"}
{"time":"28/Jul/2023:06:55:52.631", "level": "INFO", "message": "10.100.2.7:33062 for backend mysql-replicas is OK"}
{"time":"28/Jul/2023:06:55:54.309", "level": "INFO", "message": "10.100.0.16:33062 @@super_read_only: 1 Applier: ON"}
{"time":"28/Jul/2023:06:55:54.312", "level": "INFO", "message": "10.100.0.16:33062 for backend mysql-replicas is OK"}
```